lib/commons/aria/attributes.js

```diff
@@ -1,43 +1,55 @@
-/*global aria, axe, lookupTables */
+/*global aria, axe, lookupTable */
```

Comment:
We should pull these from `aria.lookupTable` instead of relying on an implicit global.